src/sokoban.hpp

```diff
@@ -1,22 +1,67 @@
 #ifndef __SOKOBAN_H__
 #define __SOKOBAN_H__
 
+#include <unordered_map>
 #include <string>
 #include <vector>
 
 class Sokoban {
+public:enum Direction {
+        U = 'U',
+        D = 'D',
+        L = 'L',
+        R = 'R'
+    };
+
+private:
+    enum Cell {
+        PLAYER = '@',
+        PLAYER_ON_GOAL = '+',
+        BOX = '$',
+        BOX_ON_GOAL = '*',
+        GOAL = '.',
+        WALL = '#',
+        EMPTY = ' '
+    };
+
+    std::unordered_map<Direction, std::vector<int>> dir_offset {
+        {Direction::U , {-1,  0}},
```

Comment:
No space after these commas

Reply:
like this: `{Direction::U ,{-1,0}},` ?

Reply:
Sorry, should have been clearer: `{Direction::U, {-1, 0}},`